build/vsts-ci.yml

```diff
@@ -49,6 +49,8 @@ phases:
     - agent.os -equals Darwin
   steps:
   # Only build native assets to avoid conflicts.
```

Comment:
This comment should go below your new script - it applies to the `./build.sh` line.